src/auth/oauth2client.ts

```diff
@@ -17,18 +17,20 @@
 import {AxiosError, AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios';
 import * as http from 'http';
 import * as querystring from 'querystring';
-
 import {PemVerifier} from './../pemverifier';
 import {BodyResponseCallback} from './../transporters';
 import {AuthClient} from './authclient';
 import {CredentialRequest, Credentials} from './credentials';
 import {LoginTicket} from './loginticket';
 
+const isString = require('lodash.isstring');
```

Comment:
Can we use import signature on this? `@sindresorhus/is` can be an alternative as well.

Reply:
Let's open this as a bug. A cherry-pick between branches shouldn't edit.

Reply:
FWIW I did try to do that :) Couldn't get it to work and decided to come back later.